### ISSUE TITLE
null is not an object (evaluating 'UpiModule.intializePayment')

### ISSUE THREAD

**Issue by amitleuva1987** [nitish24p/react-native-upi#38]
when I run my 'react native' project in 'expo go' app, I receive below error.

null is not an object (evaluating 'UpiModule.intializePayment'). below is the code.

     import React, {useState} from 'react';
     import { SafeAreaView, Text, View, Button, TextInput, StyleSheet } from 'react-native';
     import RNUpiPayment from 'react-native-upi-payment';
    
    function Checkout(){
    const text = 'Please enter a value';
    const [status, setStatus] = useState('');
    function initialize(){
        RNUpiPayment.initializePayment({
            vpa: '9033049648@ybl', // or can be john@ybl or mobileNo@upi
            payeeName: 'Amit Leuva',
            amount: '1',
            transactionRef: '123'
        },successCallback,failureCallback);
    }
    
    function successCallback(data){
        setStatus(status='success');
    }
    function failureCallback(data){
        setStatus(status='success');
    }
    return(
        <SafeAreaView>
        <View>
            <Button title='Pay now' onPress={() => initialize()} />
        </View>
        </SafeAreaView>
     );
    }
    
    export default Checkout;

Do you have alternate way to make this package work with my project?

**Comment by tejasvi** [comment]
It seems there is no working solution for iOS.

**Comment by nitish24p** [comment]
Yes this is only for android. Also expo if expo doesnt provide native modules support for now this wont work.

**Comment by Bhav2307** [comment]
same issue.

TypeError: null is not an object (evaluating 'UpiModule.intializePayment')

**Comment by nitish24p** [comment]
Please check if the module has been linked properly.

**Comment by rakeshlookings** [comment]
Do not use expo because it lacks native module support

**Comment by Rahul6694** [comment]
TypeError: null is not an object (evaluating 'UpiModule.intializePayment'

**Comment by Karthikpkapture** [comment]
Facing the same issue on react native expo  "TypeError: Cannot read property 'intializePayment' of null "

**Comment by nitish24p** [comment]
Not sure if this will work with expo, can you console.log the UpiModule value
